### ISSUE TITLE
Excercise Bug

### ISSUE THREAD

**Issue by fskarmeta** [learnpack/learnpack#63]
# What was the expected behavior?

Two issues:

Seems like the only correct output for passing the test is doing exactly the same code as the one in the unitary test. 
I did diferent versions and none of them did pass in spite of having the expected output. 
My example code here:
```
let matrixBuilder = (n) => {
    let matrix = []
    for (i = 0; i < n; i++) {
        matrix[i] = [];
        for (j= 0; j < n; j++) {
            matrix[i][j] = 1 // Math.floor(Math.random()*2) if you want random 1's and 0's
        }
    }
    return matrix
}
```

The second issue is correlated with the unitary test:  the exercise requests a "matrix of random 0's, and 1's" , thou the unitary test is filled just with ones. 

## Other details: 
1. Exercises: 22-Matrix-Builder
2. Current URL: https://feb9069d-7403-4283-9663-19551c21ee3c.ws-us02.gitpod.io
3. Repository: uknown

**Comment by ElviraQDP** [comment]
regarding your first issue, yes test can be very unflexible, we are working on that. Regarding your 2nd issue we wil fix asap
thank you!

**Comment by tommygonzaleza** [comment]
The issue has been already fixed, is the same as #151.
PR: https://github.com/4GeeksAcademy/javascript-arrays-exercises-tutorial/pull/14
